ktlint-api-consumer/build.gradle.kts

```diff
@@ -5,7 +5,7 @@ plugins {
 dependencies {
     // Any SLF4J compatible logging framework can be used. The "slf4j-simple" logging provider is configured in file
     // ktlint-api-consumer/src/main/resources/simplelogger.properties
-    runtimeOnly("org.slf4j:slf4j-simple:2.0.7")
+    runtimeOnly(libs.slf4j)
```

Comment:
Please revert this change. I intended this module to be an easy example of how an API consumer can look like. So the slf4j dependency is not meant to be used anywhere else in the project.

Maybe something similar should also be done for line 10 and 11 as well as for other modules. We might need to distinct between modules meant for publication as artifact (e.g. being a part of the Ktlint API or CLI) and other modules that are only meant for clarification. The latter group of modules should not reuse any of the common build logic but be a standalone build script. What do you think?

Reply:
cb9b94a0c80157fa858c89ece8e8357ecac50bd4

Reply:
I still we should put this dep into version catalog, we should not reuse publication logic but deps.